### PR TITLE
Make module work on Drupal 8 Beta 4

### PR DESCRIPTION
**Pull request by lauriii**

### CODE REVIEW

File: sandwich.routing.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 sandwich.build:
   path: '/sandwich'
   defaults:
-    _content: '\Drupal\sandwich\Controller\SandwichController::build'
+    _controller: '\Drupal\sandwich\Controller\SandwichController::build'
```
Comment (joelpittet) [inline sandwich.routing.yml:4]: This is a nice change:)

**Comment by joelpittet** [comment]
Thanks Lauriii!

**Comment by star-szr** [comment]
Indeed, thanks @lauriii!

**Comment by lauriii** [comment]
No problem! I will use this module in my session for a demo purposes so needed to fix it :)

**Pull request merged**